### PR TITLE
[Video] Rename pages and assets

### PR DESCRIPTION
**Pull request by Hiramiya**
Updated docs pages as part of the move towards the new service name

### CODE REVIEW

File: roadmap/video.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+
+### Feed Management  
+
+Specific authentication is required for streaming to our platform, with the ability to manage the stream keys coming soon to Dash. This will achieve the following:
 
 - Generate own named streams and key pairs via Dash
 - Ability to issue "temporary" keys and revoke these when done
 - Know exactly which streams are associated with your account
 - Know which streams are currently in use, or recently used
-- Protect the Vantibyte platform from potential abuse
```
Comment (benolot) [inline roadmap/video.md:14]: Why are we removing this line?

File: roadmap/video.md [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
-title: RTMP
+title: Video
 parent: Roadmap
 ---
 
-### Authentication  
+### Examples for other software
 
-Specific authentication will be required for streaming to our platform, with the ability to manage the stream keys via Dash. This will achieve the following:
+Currently our documentation focuses on using OBS and VLC, other options are available and examples using some other software packages will be added
```
Comment (benolot) [inline roadmap/video.md:8]: I think we should specifically mention the RTMP protocol here for SEO

File: roadmap/video.md (hunk 1 above)
Comment (Hiramiya) [reply roadmap/video.md:14]: That line was relevant to when we were _enforcing_ Authentication (that's how old this text is..) and isn't strictly related to the ability for a customer to manage their own Video keys

File: roadmap/video.md (hunk 2 above)
Comment (Hiramiya) [reply roadmap/video.md:8]: Isn't that on the Video page itself, rather than the roadmap? Also on the main website repo - should check we have it there

Comment (benolot) [reply roadmap/video.md:8]: Ah yeah true, nvm then

**Review by benolot — APPROVED** [review]

**Comment by Hiramiya** [comment]
Added some keywords on the main Video page to reference RTMP being used for ingest

**Review by benolot — APPROVED** [review]

**Pull request merged**